x/ccv/provider/keeper/grpc_query.go

```diff
@@ -141,7 +141,7 @@ func (k Keeper) GetConsumerChain(ctx sdk.Context, consumerId string) (types.Chai
 
 	metadata, err := k.GetConsumerMetadata(ctx, consumerId)
 	if err != nil {
-		return types.Chain{}, fmt.Errorf("cannot get metadata for consumer (%s): %w", consumerId, err)
+		k.Logger(ctx).Error("cannot get metadata for consumer (%s): %w", consumerId, err)
```

Comment:
Why log an error instead of returning it?

Reply:
because I preferred this behaviour from a user perspective than erroring out and not returning anything.
There's no data, so just don't show it. To me user experience is worse to not get anything because of
a chain not having  on which I'm not intereseted at all. 
But that's my preference, can undo if you don't agree and prefere the old behaviour. just let me know